docs/appendix/apis/components/arm.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Arm API"
+linkTitle: "Arm"
+weight: 20
+type: "docs"
+description: "The arm API allows you to give commands to your arm components with code instead of with the graphical interface of the Viam app"
+icon: true
+images: ["/icons/components/arm.svg"]
+---
+
+The arm API allows you to give commands to your [arm components](/components/arm/) with code instead of with the graphical interface of the [Viam app](https://app.viam.com/).
```

Comment:
As above, this feels almost like a distraction. Like, if I'm a dev, maybe I came here and never would've even considered using the GUI for prod use case--I just need an API just as I need an API for any other thing in the world.